docs/docs/guide/debugging.mdx

```diff
@@ -0,0 +1,303 @@
+
+For debugging web apps, you may use the tools you prefer or any that are provided
+by the browser of your choice. We did not discover any issues with debugging
+apps using Reanimated on web.
+
+### Summary
+
+<SummaryTable />
+
+:::caution
+
+Remember that console logs will always appear in the primary JS context as the
+`console.log` function on the UI thread is just a reference to the one from
+the JS thread.
+
+:::
+
+### Chrome Debugger
+
+<ChromeDebuggerTable />
+
+**Summary:**
+Works, but uses web implementations of functions and runs everything on the JS thread.
+
+Since the Chrome Debugger runs its own web worker, all the code is run on
+the JS thread, and it uses the JavaScript engine provided by your web
+browser (V8 in Chrome, JSC in Safari and SpiderMonkey in Firefox). This
+means that this piece of code:
+
+```js
+function runWorklet() {
+  'worklet';
+  console.log('worklet:', _WORKLET);
+}
+runOnUI(runWorklet)();
+```
+
+would output:
+
+```
+LOG: worklet: false
+```
+
+Another side effect is that Reanimated uses web implementations of all
+functions. This means that the `scrollTo` function will work
+(using the native web implementation), but the `measure`
+function will not be available, and its usage will trigger this error:
+
+```
+[reanimated.measure] method cannot be used for web or Chrome Debugger
+```
+
+You may still use the standard web version of measure as described
+[here](../api/nativeMethods/measure).
+
+Those functions that are provided by Reanimated and do not have web
+implementations won't work at all.
+An example of this behavior is the `useAnimatedSensor` hook,
+which currently only works on mobile platforms. When debugging in Chrome and using
+this hook, the following message will appear in the logs:
+
+```
+[Reanimated] useAnimatedSensor is not available on web yet.
+```
+
+But despite all of this, it is still possible to set breakpoints both in
+normal JS code and in worklets (since they run on the main JS
+thread now).
+
+### Chrome DevTools
+
+<ChromeDevToolsTable />
+
+**Summary:**
+Works and both contexts can be debugged.
+
+:::warning
+
+This is an early experimental feature.
+
+:::
+
+Worklet debugging is now available in Reanimated apps as an early experimental
+feature. We do not recommend using Chrome DevTools to debug the UI context, but
+instead try using the Flipper tool. It has fewer bugs and provides a much better
+experience.
+
+If you still want to use Chrome DevTools then follow these steps:
+
+1. Add the `patch-package` to your project and set it up to run as a `post-install` task.
+More details can be found [here](https://www.npmjs.com/package/patch-package).
+
+2. Add [this](/content/metro-inspector-proxy+0.72.1.patch) patch for `metro` and
+[this](/content/react-native-gesture-handler+2.5.0.patch) one for `react-native-gesture-handler`
+to the `patches/` folder. *Warnings about mismatched patch versions can
+be safely ignored if the patch was successfully applied*.
```

Comment:
Split this two points into separate one for `metro-inspector-proxy` (replace version with `<0.xx.yy>`) and separate for gesture handler (add comment that you can skip this step with RNGH `>=2.7.0`).